### PR TITLE
Newcdtime

### PR DESCRIPTION
**Pull request by dnadeau4**

### CODE REVIEW

**Comment by dnadeau4** [comment]
I just change the version to circleci to 3.1.0 for conda_upload.

**Review by doutriaux1 — APPROVED** [review]
While we're at it we might want to improve a bit the doc strings?

Did you change anything in `cdtimemodule.c` or is it just styling?

**Comment by durack1** [comment]
@doutriaux1 @dnadeau4 improving the docstrings would be a key improvement, as I have struggled to figure out how to use this in the past. I have some demos available if you wanted to use these in the more descriptive docstrings

**Comment by dnadeau4** [comment]
@doutriaux1 just styling.  I will work on docstring to make it more like Sphynx.  I don't think cdtime is in readthedocs yet.

**Comment by dnadeau4** [comment]
@durack1 send some demo,  I might add it to the docstring as examples.

**Comment by durack1** [comment]
@dnadeau4 the [makeCalendar function](https://github.com/durack1/durolib/blob/master/lib/durolib.py#L623-L749) and the [trimModelList function](https://github.com/durack1/durolib/blob/master/lib/durolib.py#L1008-L2577) (apologies there are some log reports in the top of this func that were being used to validation, I'll remove these on the next PR) will likely provide some real-world examples

File: Lib/compreltime.py [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+        Parameters
+        ----------
+            year:
+            month:  default = 1
+            day: default = 1
+            hour: default = 0
+            minute: default = 0
+            second: default = 0
+
+        Retuns
```
Comment (durack1) [inline Lib/compreltime.py:107]: Retuns -> Returns

File: Lib/compreltime.py [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
```
Comment (durack1) [inline Lib/compreltime.py:138]: Returns

File: Lib/compreltime.py [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+        Create a retlative time object.
+
+            relativetime(value, units)
+
+        Parameters
+        ----------
+            values:  number (can be floating point)
+            units:  [seconds,hours,days,yeears] since [year]
+
+        Retuns
```
Comment (durack1) [inline Lib/compreltime.py:216]: Returns

File: Lib/compreltime.py [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
```
Comment (durack1) [inline Lib/compreltime.py:240]: etc etc

**Comment by dnadeau4** [comment]
@durack1 Thanks!

**Comment by jypeter** [comment]
About the docstrings, would it be possible to a add in the **cdtime** related docstrings a link to the **cdtime** section of the cdms documentation? e.g. `Further information: https://cdms.readthedocs.io/en/cdmsdocsmerge/manual/cdms_3.html`. I have just had a quick look at the readthedocs page, and it matches nicely the original (and excellent) pdf of Bob Drach! The cdms documentation (v4.0) is dated November 2003 and has not aged a bit. Time flies... 
> 
> @doutriaux1 @dnadeau4 improving the docstrings would be a key improvement, as I have struggled to figure out how to use this in the past. I have some demos available if you wanted to use these in the more descriptive docstrings

**Pull request merged**